NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -41,25 +41,9 @@
     <Product id="1010">
         <ExternalId>1010</ExternalId>
         <Reviews>
-            <Review id="11" removed="false">
-                <ModerationStatus>APPROVED</ModerationStatus>
-                <UserProfileReference id="miketester112">
-                    <ExternalId>miketester112</ExternalId>
-                    <DisplayName>supermike</DisplayName>
-                    <Anonymous>false</Anonymous>
-                    <HyperlinkingEnabled>false</HyperlinkingEnabled>
-                </UserProfileReference>
-                <Title>Best product ever!</Title>
-                <ReviewText>This product truly changed my life; I don't know what I'd do without it.</ReviewText>
-                <Rating>5</Rating>
-                <IpAddress>64.94.228.137</IpAddress>
-                <Recommended>true</Recommended>
-                <UserEmailAddress>supermike@example.com</UserEmailAddress>
-                <ReviewerLocation>Austin, TX</ReviewerLocation>
-                <SubmissionTime>2012-01-23T12:59:25.000-06:00</SubmissionTime>
-                <Featured>false</Featured>
-                <DisplayLocale>en_US</DisplayLocale>
-            </Review>
+
+            <!-- basic review -->
```

Comment:
Can you add a blurb that explains the purpose of this example vs the example review below? i.e. Does this include only required elements? Only recommended elements? Most common elements?